redact-wasm/package.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "@arcjet/redact-wasm",
+  "version": "1.0.0-alpha.22",
+  "description": "Arcjet PII identification and redaction engine.",
```

```suggestion
  "description": "Arcjet sensitive information redaction detection engine",
```
Aligm but I remove the periods on these